Google style expects *args not args

```py
def test(*args, **kwargs):
    """A summary description.

    Args:
        *args: google-style expects *args, not args.
        **kwargs: google-style expects **kwargs, not kwargs.

    Note:
        https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html
    """
```

> test.py:1:1: D417 Missing argument descriptions in the docstring: `**kwargs`, `*args`

From https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html:

>     If ``*args`` or ``**kwargs`` are accepted,
>     they should be listed as ``*args`` and ``**kwargs``.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -36,3 +36,6 @@ urls = { repository = "https://github.com/charliermarsh/ruff-lsp" }
 [tool.maturin]
 bindings = "bin"
 strip = true
+
+[tool.ruff.pydocstyle]
+convention = "google"

--- a/resources/test/fixtures/pydocstyle/D417.py
+++ b/resources/test/fixtures/pydocstyle/D417.py
@@ -1,5 +1,5 @@
 def f(x, y, z):
-    """Do f.
+    """Do something.
 
     Args:
         x: the value
@@ -12,7 +12,7 @@ def f(x, y, z):
 
 
 def f(x, y, z):
-    """Do f.
+    """Do something.
 
     Args:
         x:
@@ -25,7 +25,7 @@ def f(x, y, z):
 
 
 def f(x, y, z):
-    """Do f.
+    """Do something.
 
     Args:
         x:
@@ -37,7 +37,7 @@ def f(x, y, z):
 
 
 def f(x, y, z):
-    """Do f.
+    """Do something.
 
     Args:
         x: the value def
@@ -50,7 +50,7 @@ def f(x, y, z):
 
 
 def f(x, y, z):
-    """Do f.
+    """Do something.
 
     Args:
         x: the value
@@ -63,7 +63,7 @@ def f(x, y, z):
 
 
 def f(x, y, z):
-    """Do g.
+    """Do something.
 
     Args:
         x: the value
@@ -75,10 +75,41 @@ def f(x, y, z):
 
 
 def f(x, y, z):
-    """Do h.
+    """Do something.
 
     Args:
         x: the value
         z: A final argument
+    """
+    return x
+
+
+def f(x, *args, **kwargs):
+    """Do something.
+
+    Args:
+        x: the value
+        *args: variable arguments
+        **kwargs: keyword arguments
+    """
+    return x
+
+
+def f(x, *args, **kwargs):
+    """Do something.
+
+    Args:
+        *args: variable arguments
+        **kwargs: keyword arguments
+    """
+    return x
+
+
+def f(x, *args, **kwargs):
+    """Do something.
+
+    Args:
+        x: the value
+        **kwargs: keyword arguments
     """
     return x

--- a/src/pydocstyle/plugins.rs
+++ b/src/pydocstyle/plugins.rs
@@ -1413,7 +1413,7 @@ fn missing_args(checker: &mut Checker, docstring: &Docstring, docstrings_args: &
 
 // See: `GOOGLE_ARGS_REGEX` in `pydocstyle/checker.py`.
 static GOOGLE_ARGS_REGEX: Lazy<Regex> =
-    Lazy::new(|| Regex::new(r"^\s*(\w+)\s*(\(.*?\))?\s*:.+").unwrap());
+    Lazy::new(|| Regex::new(r"^\s*(\*?\*?\w+)\s*(\(.*?\))?\s*:.+").unwrap());
 
 fn args_section(checker: &mut Checker, docstring: &Docstring, context: &SectionContext) {
     let mut matches = Vec::new();

